src/main/java/org/folio/circulation/domain/policy/LostItemPolicy.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.circulation.domain.policy;
+
+import io.vertx.core.json.JsonObject;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+
```

Comment:
Please remove this unnecessary blank line

Reply:
Removed